packages/x-codemod/README.md

```diff
@@ -86,7 +86,20 @@ npx @mui/x-codemod v6.0.0/pickers/preset-safe <path|folder>
 
 The list includes these transformers
 
-NO CODEMOD ADDED YET
+- [`rename-default-calendar-month-reference-date`](#rename-default-calendar-month-reference-date)
```

Comment:
Could you apply this change and the additional diff to fix the inconsistencies? 🤔 
```suggestion
- [`rename-components-to-slots-pickers`](#rename-components-to-slots-pickers)
- [`rename-default-calendar-month-reference-date`](#rename-default-calendar-month-reference-date)
```

```diff
diff --git a/packages/x-codemod/README.md b/packages/x-codemod/README.md
index 4a5bf4f7c..62829228d 100644
--- a/packages/x-codemod/README.md
+++ b/packages/x-codemod/README.md
@@ -76,12 +76,12 @@ The corresponding sub-sections are listed below
 
 ### Pickers codemods
 
-#### `preset-safe` for pickers v6.0.0
+#### `preset-safe` for pickers v7.0.0
 
 The `preset-safe` codemods for pickers.
 
-npx @mui/x-codemod v6.0.0/pickers/preset-safe <path|folder>
+npx @mui/x-codemod v7.0.0/pickers/preset-safe <path|folder>
 
 The list includes these transformers
@@ -95,7 +95,7 @@ NO CODEMOD ADDED YET
 The `preset-safe` codemods for data grid.
 
-npx @mui/x-codemod v6.0.0/data-grid/preset-safe <path|folder>
+npx @mui/x-codemod v7.0.0/data-grid/preset-safe <path|folder>
 
 The list includes these transformers
```

Reply:
For the component to slots prop I'd like to duplicate the codemod in a follow up PR because right now it's in v7 preset but can be applied using `v7.0.0/pickers/rename-components-to-slots-pickers` (same on the grid I guess).

Diff applied